SpinCore_pp/ppg/generic.py

```diff
@@ -117,8 +118,9 @@ def generic(
         configureTX(adcOffset, carrierFreq_MHz, tx_phases, amplitude, nPoints)
         run_scans_time_list.append(time.time())
         run_scans_names.append("configure Rx")
-        check = configureRX(SW_kHz, nPoints, nScans, nEchoes, nPhaseSteps)
-        assert acq_time_ms == check
+        check = round(configureRX(SW_kHz, nPoints, nScans, nEchoes, int(nPhaseSteps)),1)
+        #there is some very small difference so we round to the first decimal 
+        assert round(acq_time_ms,1) == check
```

Comment:
I think that my previous comment got accidentally marked as resolved --> this is still problematic

Reply:
I am rolling back but issue still remains that if the fed acq_time_ms is not within 0.000000001 of the output of configureRX it throws an error hence the round